betterrolls-swade2/templates/actions_partial.html

```diff
@@ -9,7 +9,7 @@
                 </span>
             {{/each}}
         {{else}}
-            {{ localize "BRSW.NoActionsSelected" }}
+            {{ this.no_actions_message }}
```

Comment:
This change is unnecessary. If it's a string ID it will localize it and if not it will just print the string. No need to handle this as a special case.

Reply:
Added localize but kept this.no_actions_message since without it we won't get the setting.